src/link.js

```diff
@@ -113,11 +93,27 @@ module.exports = function link(config, args, callback) {
   };
 
   const tasks = dependencies.map((dependency) => (next) => {
-    const prelink = makeHook(dependency, 'prelink');
-    const postlink = makeHook(dependency, 'postlink');
-    const link = makeLink(dependency);
+    const queue = [];
+
+    var prelink;
+    var postlink;
+
+    if (dependency.config.commands) {
+      prelink = dependency.config.commands.prelink;
+      postlink = dependency.config.commands.postlink;
+    }
```

Comment:
What happens when these are not defined? Is `asycn.waterfall` capable of handling undefined `prelink`, `postlink`? I think commands are **always** defined so we should check for presence of prelink/postlink instead